CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -70,12 +70,12 @@ public let ATTRIBUTE_NODES: [Node] = [
             kind: .node(kind: .stringLiteralExpr)
           ),
           Child(
-            name: "Availability",
-            kind: .node(kind: .availabilitySpecList)
+            name: "AvailabilityArguments",
```

Comment:
I have renamed it to maintain consistency with other cases.

Reply:
We recently introduced `deprecatedName` on `Child`. 
This would generate the needed variables, initialisers etc.

Can you try to use that re-run the code generator?

Reply:
I tried using the `deprecatedName` like this, but there was no change. 
```swift
  Child(
    name: "ArgumentList",
    deprecatedName: "Availability",
    kind: .node(kind: .tupleExprElementList)
  )
```
The code generated from this section is `nodeChoices` (enum with associated values), so I thought it wasn't working 🤔 
Could there be something I missed?

Reply:
Have you committed the changes? 
Because it uses the HEAD of the branch. 

You can run swift package resolve to get the latest.

Reply:
You might also need to try removing `CodeGeneration/Package.resolved` and `CodeGeneration/.build`. We are having some issues with the `HEAD` reference at the moment 😢

Reply:
I tried both
- committing the changes and resolving the package
- deleting `CodeGeneration/Package.resolved` and `CodeGeneration/.build`

but it still doesn't work 😭 
The strange thing is that modifying the `deprecatedName` in other parts, not this section, works fine. (e.g. deprecatedName at line 176)
Is there any way to debug this?

Reply:
Oh, I see! Thank you all for your help! 😊
May I ask one more question?
If I were to write a compatibility layer for an Enum case with associated values like this, how could I do it? 
```swift
    case `availability`(TupleExprElementListSyntax)
```
I know that I can write a `static var` for cases without associated values, but I'm unsure about cases with associated values as I couldn't find any examples.

Reply:
Aha, there is no perfect way to maintain compatibility with switch-case. Thank you for letting me know 🙇‍♂️